docs/ci-cd.md

```diff
@@ -0,0 +1,59 @@
+We run various quality checks at build time to ensure that the documentation is maintainable.
+
+Some of the checks can be manually marked as approved using labels:
+
+* `ci:manual-approve:link-validation` - mark link checking as successful. Useful when Netlify returns an `HTTP 400` error and the links are validated manually.
```

Comment:
```suggestion

## Skipping checks or processes

Some of the checks can be manually adjusted depending on a PR's needs:

* Set labels:
  * `ci:manual-approve:link-validation` label - mark link checking as successful. Useful when Netlify returns an `HTTP 400` error and the links are validated manually.
  * `skip-changelog` - mark an item that you don't want to appear in the weekly [docs changelog](https://github.com/Kong/docs.konghq.com/blob/main/changelog.md).

* Set `[skip ci]` in the title of a PR to skip Netlify preview generation. This is useful for PRs that don't have any visual effect on the docs site, such as editing the repo README.
```

I don't really know what to title this section, but it should be something that summarizes that these settings change something in our CI processes and checks.